pages/mesosphere/dcos/services/dse/3.1.0-6.7.6/release-notes/index.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+excerpt: Release Notes for DSE v. 3.1.0-6.7.6
+title: Release Notes
+menuWeight: 10
+model: /mesosphere/dcos/services/dse/data.yml
+render: mustache
+---
+
+# Version 3.1.0-6.7.6
+
+## Updates
+- Upgraded {{ model.techShortName }} to version 6.7.6 and {{ model.techOpsName }} to version 6.7.6.
+- Upgraded SDK to 0.57.0.post0. For more information see the SDK's Release Notes for: 
+  - [0.57.0.post0](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.0.post0)
+  - [0.57.0](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.0)
+  - [0.56.3](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.3)
+  - [0.56.2](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.2)
+  - [0.56.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.1)
+  - [0.56.0](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.0)
+  - [0.55.5](https://github.com/mesosphere/dcos-commons/releases/tag/0.55.5)
+
+## New Features
+- Added support for DSS [volume profiles](/mesosphere/dcos/services/{{ model.serviceName }}/latest/configuration/#using-volume-profiles).
+- Added [custom domain](/mesosphere/dcos/services/{{ model.serviceName }}/latest/security/#forwarding-dns-and-custom-domain) support.
+
+## Improvements
+- Removed non-HTTPS resources and references by replacing them with HTTPS versions.
+- Added more directories to the PATH (resources/cassandra/bin, resources/cassandra/tools/bin).
+- Added descriptions for configuration parameters.
+- Updated UI URLs to redirect to correct reference doc pages.
+- Excluded {{ model.techOpsName }} keyspace from list of keyspaces managed by Repair Services
+- Improved Accurate Scheduler Plan status in the UI.
+
+## Bug Fixes
+- Used correct YAML markup for TPC-related configurations: TPC CORES, TPC IO CORES, IO GLOBAL QUEUE DEPTH.
+- DSEFS could be enabled independently from DSE Analytics.
+- Ignore error code of `nodetool` commands; stops command from running indefinitely.
+- Fixed issue where OpsCenter event-plugins were not being saved and were getting lost on restart. Now, alert-sending configurations are persisted between OpsCenter restarts.
+- Fixed OpsCenter UI, which was showing a gray ring after a restart.
+
+## Upgrading your cluster from {{ model.techShortName }} 5.1.10 to 6.7.6
+Due to the complexity of upgrading to {{ model.techShortName }} 6.7, we strongly advise that you attempt the upgrade on a test cluster before upgrading in your production environment. See the [update section](/mesosphere/dcos/services/dse/3.1.0-6.7.6/updates/) for specific instructions.
+
```

Comment:
You didn't want to include any of the release notes from 6.7.2?

Reply:
Hi Sam, I would still prefer that but it been removed by @SarahStegall in https://github.com/mesosphere/dcos-docs-site/pull/2554/commits/92ec9ebcb5b6eb8bee61124afbf1b3a701329eb3